cpp/src/reductions/reductions.cpp

```diff
@@ -88,19 +88,19 @@ struct reduce_dispatch_functor {
                                         stream,
                                         mr);
         return get_element(*col_ptr, 0, mr);
-      } break;
+      }
       case aggregation::NUNIQUE: {
         auto nunique_agg = static_cast<nunique_aggregation const *>(agg.get());
         return make_fixed_width_scalar(
           detail::distinct_count(
             col, nunique_agg->_null_handling, nan_policy::NAN_IS_VALID, stream.value()),
           stream.value(),
           mr);
-      } break;
+      }
```

Comment:
Isn't compiler throwing a warning here for leaving out `break;`?

Reply:
I don't think so? all of the cases have a `return` - which makes `break` unnecessary (IIUC).